.changeset/short-jokes-applaud.md

```diff
@@ -0,0 +1,13 @@
+---
+'@backstage/core-components': minor
+'@backstage/plugin-catalog': minor
+'@backstage/plugin-catalog-react': minor
```

Comment:
could you break this apart into three changesets? or at least two, one for core components by itself. The core-components one can actually be patch level instead of minor too, since it's on v0.x still.